presto-product-tests/conf/presto/etc/multinode-worker.properties

```diff
@@ -13,5 +13,4 @@ http-server.http.port=8081
 query.max-memory=1GB
 query.max-memory-per-node=512MB
 query.max-total-memory-per-node=1GB
-discovery-server.enabled=false
```

Comment:
@electrum after this change, does it become invalid to have `discovery-server.enabled` in worker config.properties?

Ideally, coordinator's and worker's config.properties should differ only with `coordinator=..`. Of course, if you have everything automated, it doesn't matter that much, but i observe people oftentimes do not have proper automation in place. Also, even with some kind of automation (like presto-admin) if you have separate config.properties for coordinator and workers, they will go out of sync.

Case observed today -- `query.max-memory=` being set in worker properties instead of on the coordinator. I would like to reply "hey, you should set this on the coordinator. As a rule, keep the config files in sync, differing only by the `coordinator=` flag"

Reply:
or, maybe, it's already not possible to have almost identical config files (unless you use defaults 99% time). @electrum ?

Reply:
There are often many differences between coordinator and worker configurations.